src/server/zset_family_test.cc

```diff
@@ -523,6 +523,16 @@ TEST_F(ZSetFamilyTest, ZInterStore) {
   EXPECT_THAT(resp.GetVec(), ElementsAre("b", "3"));
 }
 
+TEST_F(ZSetFamilyTest, ZInter) {
+  EXPECT_EQ(2, CheckedInt({"zadd", "z1", "1", "one", "2", "two"}));
+  EXPECT_EQ(3, CheckedInt({"zadd", "z2", "1", "one", "2", "two", "3", "three"}));
+  RespExpr resp;
+
+  resp = Run({"zinter", "2", "z1", "z2"});
+  EXPECT_THAT(resp, ArrLen(2));
+  EXPECT_THAT(resp.GetVec(), ElementsAre("one", "two"));
```

Comment:
```suggestion
  EXPECT_THAT(Run({"zinter", "2", "z1", "z2"}), RespArray(ElementsAre("one", "two")));
```

Reply:
It looks like this fails in CI, and the reason is that you expect the order to be `'one', 'two'`, while the returned array is `'two', 'one'`.
I'm not sure if `ZINTER` should sort elements before replying, can you please verify? If so, please sort, otherwise, you could use `UnorderedElementsAre()` instead.

Reply:
I read the redis documentation on zinter.
Since there is no mention of key order, I don't think sorting is necessary.
So instead of using ElementsAre, I will run the test with UnorderedElementsAre.

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2164/commits/bbd02f13773e27f1398f88e30cacf94d6cc6076f